sql/item_mods.sql

```diff
@@ -53905,7 +53905,11 @@ INSERT INTO `item_mods` VALUES (25718,165,3);   -- CRITHITRATE: 3
 INSERT INTO `item_mods` VALUES (25718,384,400); -- HASTE_GEAR: 400
 
 -- Jubilee Shirt
-INSERT INTO `item_mods` VALUES (25722,1,1); -- DEF: 1
+INSERT INTO `item_mods` VALUES (25722,1,1);     -- DEF: 1
+
+-- Kupo Suit
+INSERT INTO `item_mods` VALUES (25726,169,18);  -- Movement speed: +18%
```

Comment:
Please use the exact enum values in comments

Reply:
Would that be MOVEMENT_SPEED? I'm coming up empty-handed searching through the repo for examples...

![image](https://user-images.githubusercontent.com/65316697/177193943-2549ed82-c118-4d2d-9648-8ef351afc5a5.png)

Reply:
You're referencing item_latents in regards to item_mods.sql.  Please check the updated format, and it _must_ match the enum name and value for what you're adding.

Reply:
Nevermind, think I found it...